### PR TITLE
feat: bump react-pro-sidebar and refactor

### PR DESCRIPTION
**Pull request by dohaki**
Updates sidebar lib hoping that this will fix https://linear.app/uma/issue/ACX-1275/landing-page-menu-button-doesnt-respond-on-click.

Initial tests confirm a performance boost

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "react-dom": "^17.0.2",
     "react-feather": "^2.0.9",
     "react-lottie": "^1.2.3",
-    "react-pro-sidebar": "^0.7.1",
+    "react-pro-sidebar": "^1.1.0-alpha.1",
```
Comment (james-a-morris) [inline package.json:41]: Should we be bumping to an alpha build for prod?

Comment (dohaki) [reply package.json:41]: But yea I guess pinning to `1.0.0` makes sense.

Comment (dohaki) [reply package.json:41]: Ok, I would actually prefer the 1.1.0 alpha version 😅 It has a cleaner API and doesn't require us to wrap the app within another provider. Wdyt?

Comment (james-a-morris) [reply package.json:41]: Yeah no worries!

File: src/components/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -1,132 +1,166 @@
...
-              }
+    <ReactProSidebar
+      onBackdropClick={onClickOverlay}
+      toggled={openSidebar}
+      breakPoint="all"
+      width={sidebarWidth}
+      rootStyles={{
+        zIndex: "3000 !important",
+        // @ts-ignore
+        direction: "ltr !important", // hack to display on the right side
```
Comment (james-a-morris) [inline src/components/Sidebar/Sidebar.tsx:70]: Can we wrap this with a styles wrapper?

**Review by james-a-morris — APPROVED** [review]
LGTM - 2 nits. I ran it in preview and it looks good

File: src/components/Sidebar/Sidebar.tsx (hunk 2 above)
Comment (dohaki) [reply src/components/Sidebar/Sidebar.tsx]: done https://github.com/across-protocol/frontend-v2/pull/843/commits/97870c4cbe8a38fa0bb4ac1edbb7c667f0390a78

**Pull request merged**